phys/module_cumulus_driver.F

```diff
@@ -1057,9 +1057,10 @@ SUBROUTINE cumulus_driver(grid                                     &
                ,ZOL=zol,WSTAR=wstar,UST=ust,PBLH=pblh           & !ckay
                ,AEROCU=aerocu,NO_SRC_TYPES_CU=no_src_types_cu   &
                ,AERCU_FCT=aercu_fct,AERCU_OPT=aercu_opt        &
-               ,EFCS=EFCS,EFIS=EFIS,EFSS=EFSS)
-#endif
-
+               ,EFCS=EFCS,EFIS=EFIS,EFSS=EFSS                   &
+               ,RUCUTEN=RUCUTEN,RVCUTEN=RVCUTEN,XLAND=XLAND)       !JTR
+#endif  
+           print *,'here'     
```

Comment:
@alapaty-epa 
Kiran,
Zap this debug statement

Reply:
@davegill This is done.